assets/src/dashboard/components/cardGrid/index.js

```diff
@@ -23,10 +23,10 @@ import styled from 'styled-components';
 /**
  * Internal dependencies
  */
-import usePagePreviewSize from '../../utils/usePagePreviewSize';
+import { PageSizePropType } from '../../types';
 
-const DashboardGrid = styled.div`
-  ${({ columnHeight, columnWidth, theme }) => `
+const DashboardGrid = styled.div(
+  ({ columnHeight, columnWidth, theme }) => `
```

Comment:
I opted to take out `?.` optionals here, the pageSize is _always_ going to be passed through. And if it's not, this value needs to originate higher up so that the other places pageSize is getting used from the same context consumer are in sync. I had added the check here earlier when testing out storybook to get to something else and missed this. Sorry about that.